include/charge_controller/ChargeController.hpp

```diff
@@ -33,9 +51,19 @@ class ChargeController {
         FAULT
     };
 
-    ChargeController(BMSManager bms, LCDDisplay& display, IO::GPIO& relay);
+    static const uint32_t CHARGER_STATUS_CAN_ID = 0x18FF50E5;
```

Comment:
Could be constexpred here and would match up with the conventions in other parts of your code.